qupath-extension-openslide/src/main/java/qupath/lib/images/servers/openslide/jna/OpenSlideLoader.java

```diff
@@ -69,6 +69,19 @@ public static synchronized boolean tryToLoad(String... searchPath) throws Unsati
 
     private static synchronized OpenSlideJNA tryToLoadJnaInstance(String... searchPath) {
         String jnaPath = System.getProperty("jna.library.path", null);
+        if (searchPath.length == 0 || (searchPath.length == 1 && searchPath[0].isEmpty())) {
+            logger.info("No search path provided; trying to load OpenSlide from packaged jar");
+            try {
+                File openslideFile = Native.extractFromResourcePath("openslide");
+                return Native.load(openslideFile.getAbsolutePath(), OpenSlideJNA.class);
+            } catch (IOException e) {
+                logger.error("Failed to load OpenSlide from packaged jar");
+                if (jnaPath == null)
```

Comment:
Unless I've missed something, I don't think there's any need for that here, since `jna.library.path` hasn't been modified.